S3/Redshift Support

iter-io opened the pull request:
Please let me know if including any of this code in the repo might be of interest.  In that case I can merge the latest changes into this branch and write some documentation.

medvedev1088 commented on dags/load_dag_redshift.py:
```diff
@@ -0,0 +1,112 @@
+
+    if file_format == 'csv':
+        sql = """
+            COPY {schema}.{table}
```
To make this operation idempotent you may need to delete the records for the current `ds` before loading the file.

iter-io replied:
@medvedev1088 - Addressed by [bbfb7684f37337363db0894a7b8e65c6488f4a22](https://github.com/iter-io/ethereum-etl-airflow/commit/bbfb7684f37337363db0894a7b8e65c6488f4a22)

medvedev1088 commented on dags/load_dag_redshift.py:
```diff
@@ -0,0 +1,112 @@
+    pg_hook.run(formatted_sql)
+
+
+def add_load_tasks(task, file_format, allow_quoted_newlines=False):
```
Seems `allow_quoted_newlines` is not used.

iter-io replied:
@medvedev1088 - Addressed by [bbfb7684f37337363db0894a7b8e65c6488f4a22](https://github.com/iter-io/ethereum-etl-airflow/commit/bbfb7684f37337363db0894a7b8e65c6488f4a22)

iter-io commented:
@medvedev1088 - I addressed your comments above.  I also updated the export DAGs to use `build_export_dag.py` and created a new helper called build_load_dag_redshift.

medvedev1088 commented on dags/variables.py:
```diff
@@ -62,6 +66,23 @@ def read_load_dag_vars(var_prefix, **kwargs):
+    load_start_date = read_var('load_start_date', var_prefix, False, **kwargs)
+    if load_start_date is not None:
+        load_start_date = datetime.strptime(load_start_date, '%Y-%m-%d')
+        var_prefix['load_start_date'] = load_start_date
```
I had a bug in my code. It should be `vars` instead of `var_prefix` here.

iter-io replied:
@medvedev1088 - This is what you were referring to, right?

https://github.com/iter-io/ethereum-etl-airflow/commit/54120f414a6562b7a81be06e5ed112140f3c4134

iter-io commented:
@medvedev1088 - I'm going to thoroughly test the most recent commit tonight on my AWS setup.  But I won't be able to test the google functionality.

iter-io commented:
@medvedev1088 - I'm getting this error now:

`ModuleNotFoundError: No module named 'google.storage'`

Not sure how to install all dependencies for this project.  I think I originally figured it out via trial and error.  I expect other people will get errors from my code because of not having the boto3 module installed.
